install.bat

```diff
@@ -820,24 +822,24 @@ site_config.LUAROCKS_MD5CHECKER=[[md5sum]]
 if FORCE_CONFIG then
 	f:write("site_config.LUAROCKS_FORCE_CONFIG=true\n")
 end
-if exists(vars.LUADIR.."\\luarocks\\site_config.lua.bak") then
-	for line in io.lines(vars.LUADIR.."\\luarocks\\site_config.lua.bak", "r") do
+if exists(vars.LUADIR.."\\luarocks\\"..site_config..".lua.bak") then
+	for line in io.lines(vars.LUADIR.."\\luarocks\\"..site_config..".lua.bak", "r") do
 		f:write(line)
 		f:write("\n")
 	end
-	exec(S[[DEL /F /Q "$LUADIR\luarocks\site_config.lua.bak"]])
+	exec(S([[DEL /F /Q "$LUADIR\luarocks\]]..site_config..[[.lua.bak"]]))
 end
 f:write("return site_config\n")
 f:close()
-print(S[[Created LuaRocks site-config file: $LUADIR\luarocks\site_config.lua]])
+print(S([[Created LuaRocks site-config file: $LUADIR\luarocks\]]..site_config..[[.lua]]))
 
 -- create config file
-vars.CONFIG_FILE = vars.SYSCONFDIR.."\\config.lua"
+vars.CONFIG_FILE = vars.SYSCONFDIR.."\\"..vars.SYSCONFFILENAME
 if not exists(vars.SYSCONFDIR) then
 	mkdir(vars.SYSCONFDIR)
 end
 if exists(vars.CONFIG_FILE) then
-	local nname = backup(vars.CONFIG_FILE, "config.bak")
+	local nname = backup(vars.CONFIG_FILE, S"config-$LUA_VERSION.bak")

```

Comment:
Wouldn't it be better to just use
```lua
local nname = backup(vars.CONFIG_FILE, vars.SYSCONFFILENAME..".bak")
````
so you don't have to keep both names format in sync by hand?